templates/lobby.html

```diff
@@ -758,6 +770,10 @@
             applyRounds(parsed.data.round, maxRounds);
             applyPlayers(parsed.data.players);
 
+            //Show additional dialog, that another user (drawer) is choosing a word
+            drawerNameSpan.innerHTML = drawerName;
+            waitingDialog.style.visibility = "visible";
```

Comment:
This is not enough, it should be handled on `ready` as well, as a ready event is sent upon reconnect (F5) in which scenario might have not been chosen yet.